src/templates/_partials/_accordion.twig

```diff
@@ -0,0 +1,26 @@
+{#
+
+Accordion partial component
+Usage:
+
+{% include "_partials/_accordion.twig" with {
+  heading: "Accordion Heading",
+  contents: "<p>Your Markup</p>"
+} %}
+
+#}
+
+{% if heading is defined and heading and contents is defined and contents %}
+  <div class='accordion'>
+    <button
+      type='button'
+      aria-expanded="false"
+      data-toggle-control>
+      <span class='plus-circle-icon flex-none'></span>
+      {{ heading }}
+    </button>
+    <div class='accordion-content'>
+      {{ contents }}
+    </div>
+  </div>
+{% endif %}
```

Comment:
Make sure to create a `_accordions.twig` block that will render multiple accordions together.